pages/docs/tutorials/validate-events.md

```diff
@@ -0,0 +1,72 @@
+---
+title: messages/events validation

```

Comment:
I guess we can just stick with one name, `Messages` (upper case) in all documents as they reflect AsyncAPI spec and `message` property that is used there to describe message/event

Reply:
Love this.

Reply:
On that note, let's make sure we update the **FILE NAME** in this PR to `message-validation`.